examples/source/0.introduction/Banner_Ad.html

```diff
@@ -45,9 +45,9 @@
   </a>
 
   <!--
-   We include a tracking pixel to record that the ad was rendered using [amp-pixel](https://www.ampproject.org/docs/reference/components/amp-pixel).
+   We include a tracking pixel to record that the ad was rendered using [amp-pixel]({{g.doc('/content/amp-dev/documentation/components/reference/amp-pixel.md', locale=doc.locale).url.path}}).
 
-  <p>Alternatively one could make use of [amp-analytics](https://www.ampproject.org/docs/reference/extended/amp-analytics.html) to enable much more comprenhensive measure activity.</p>
+  <p>Alternatively one could make use of [amp-analytics]({{g.doc('/content/amp-dev/documentation/components/reference/amp-analytics.md', locale=doc.locale).url.path}}) to enable much more comprenhensive measure activity.</p>
```

Comment:
`components/reference` has changed to `components`

Reply:
the file path here is correct, only the external, user facing links changed